catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.security.xml

```diff
@@ -21,14 +21,12 @@
         <AD id="systemUserAttribute"
             name="System User Attribute"
             description="The name of the attribute to determine the system user."
-            required="true"
```

Comment:
❓ Why are these no longer required?

Reply:
Required is the default state when defining new AD tags.